scanomatic/io/rpc_client.py

```diff
@@ -77,25 +71,24 @@ def __init__(self, host, port, userID=None, log_level=None):
         self.host = host
         self.port = port
 
-    def __getattr__(self, key):
-
-        if key == 'launch_local':
-            if self.online is False and self.local:
-                self._logger.info("Launching new local server")
-                return lambda: Popen(["scan-o-matic_server"])
-            else:
-                return lambda: self._logger.warning(
-                    "Can't launch because server is {0}".format(
-                        ['not local', 'online'][self.online],
-                    )
+    def launch_local(self) -> None:
+        if self.online is False and self.local:
+            self._logger.info("Launching new local server")
+            Popen(["scan-o-matic_server"])
+        else:
+            self._logger.warning(
+                "Can't launch because server is {0}".format(
+                    ['not local', 'online'][self.online],
                 )
+            )
 
-        elif key in self._allowedMethods() and self._client is not None:
+    def __getattr__(self, key):
+        if key in self._allowedMethods() and self._client is not None:
```

Comment:
Since you did `s/userID/user_id/` you may want to consider doing the same to `allowedMethods` and userIDdecorator` too.

Reply:
I'll do that in the next PR that is almost ready.

Reply:
Tanoshimi.